nautobot/utilities/tests/test_templatetags_helpers.py

```diff
@@ -49,9 +49,11 @@ def test_render_json(self):
         self.assertEqual(
             render_json({"first": [1, 2, 3]}), '{\n    "first": [\n        1,\n        2,\n        3\n    ]\n}'
         )
+        self.assertEqual('"I am UTF-8! 💩"', render_json("I am UTF-8! 💩"))
 
     def test_render_yaml(self):
         self.assertEqual(render_yaml({"first": [1, 2, 3]}), "first:\n- 1\n- 2\n- 3\n")
+        self.assertEqual("I am UTF-8! 💩\n...\n", render_yaml("I am UTF-8! 💩"))
```

Comment:
1. We should invert the assertion check for consistency. Per Django docs it's often written as `self.assertEqual(what_was_returned, expected_to_be_returned)`
2. As well, we should put the string in a dictionary of some sort because it is very rare we would pass just a string into the `render_yaml` function (hence the `...` being outputted)